src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -60,9 +61,13 @@ internal abstract partial class AbstractNavigateToSearchService
                     // use the last computed results we have for that project.  If so, it can
                     // be much faster to reuse and filter that result than to compute it from
                     // scratch.
+#if true
                     var task = searchDocument != null
                         ? ComputeSearchResultsAsync(project, searchDocument, nameMatcher, containerMatcherOpt, nameMatches, containerMatches, cancellationToken)
                         : TryFilterPreviousSearchResultsAsync(project, searchDocument, pattern, nameMatcher, containerMatcherOpt, nameMatches, containerMatches, cancellationToken);
+#else
```

Comment:
Helper code for testing purposes.  Allows avoiding the cached results when possible.